tests/cli/test_compose_sanity.sh

```diff
@@ -50,7 +50,7 @@ rlJournalStart
     rlPhaseEnd
 
     rlPhaseStartTest "compose start again"
-        UUID=`$CLI --test=2 compose start example-http-server tar`
+        UUID=`$CLI compose start example-http-server tar`
```

Comment:
That's a very good point. I was also concerned about being sure that anaconda is already running, but it seems that it always waits at least those 20 seconds which I believe is enough, but I can't be sure. 

The best way to tell if anaconda is running should be by `ps aux | egrep '[a]naconda'` that will return `0` only if anaconda is really running. 

WRT portability, what we could do is add a check for backend and not run this anaconda check if it's osbuild-composer after https://github.com/weldr/lorax/pull/917 is merged.

Reply:
I added a commit with the proposed change. I can see I have wrong indentation, I'll fix that along with any additional change or when squashing the commits.

Reply:
Looks better now, adjust the spaces and we should be ready for merge.

Reply:
> @bcl can you give us some more info here? In particular is it possible for the compose status to be RUNNING but anaconda.pid to have not been created, meaning when we cancel we can't be sure that cleanup has been executed ?

RUNNING should always mean anaconda is running. There is a *very* small amount of time between writing the status and starting anaconda where this isn't true. If running anaconda fails for any reason it should raise and exception and the status is changed to FAILED. If an exception isn't raised then it will write FINISHED.

The code for this behavior is here - https://github.com/weldr/lorax/blob/master/src/pylorax/api/queue.py#L161